dev/minimal.Dockerfile

```diff
@@ -0,0 +1,25 @@
+FROM nginx:1.15
+
+COPY nginx.conf /etc/nginx/nginx.conf
+COPY nginx.montagu.conf /etc/nginx/conf.d/montagu.conf.template
+COPY index.html /usr/share/nginx/html/index.html
+COPY resources /usr/share/nginx/html/resources
+
+# Copy third party javascript from npm modules
+ENV THIRDPARTY_JS_PATH /usr/share/nginx/html/resources/js/third_party/
+COPY node_modules/pako/dist/pako.min.js $THIRDPARTY_JS_PATH
+COPY node_modules/vue/dist/vue.min.js $THIRDPARTY_JS_PATH
+COPY node_modules/jwt-decode/build/jwt-decode.min.js $THIRDPARTY_JS_PATH
+COPY node_modules/jquery/dist/jquery.min.js $THIRDPARTY_JS_PATH
+
+# Copy third party css from npm modules
+COPY node_modules/bootstrap/dist/css/bootstrap.min.css /usr/share/nginx/html/resources/css/third_party/
+
+RUN rm /etc/nginx/conf.d/default.conf
+
+COPY secrets /etc/montagu/proxy/.
+
+WORKDIR /app
+COPY entrypoint.sh .
```

Comment:
where is this file at the moment? I don't see how this can build, and yet it clearly is

Reply:
The entrypoint script? It's coming from the build container, this is a dockerfile that gets built from inside another one :see_no_evil: